CMakeLists.txt

```diff
@@ -314,10 +314,10 @@ endif()
 
 option(WITH_BLUEFS "libbluefs library" OFF)
 
-option(WITH_QATLIB "Enable QAT with qatlib" OFF)
+option(WITH_QATLIB "Enable QAT with qatlib" ON)
 option(WITH_SYSTEM_QATLIB "Use system packages for qatlib" OFF)
 option(WITH_QATDRV "Enable QAT with out-of-tree driver" OFF)
-option(WITH_QATZIP "Enable QATzip" OFF)
+option(WITH_QATZIP "Enable QATzip" ON)
```

Comment:
does this deserve a line in the release note? i am not sure about how important this is though.

Reply:
good idea, added a release note